bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/impl/BpmnModelConstants.java

```diff
@@ -50,6 +52,18 @@ public final class BpmnModelConstants {
   /** Zeebe namespace */
   public static final String ZEEBE_NS = "http://camunda.org/schema/zeebe/1.0";
 
+  /** Zeebe namespace */
+  public static final String MODELER_NS = "http://camunda.org/schema/modeler/1.0";
+
+  /** Zeebe version */
+  public static final String ZEEBE_VERSION = VersionUtil.getVersion();
+
+  /** BPMN Exporter */
+  public static final String BPMN_EXPORTER = "Zeebe";
```

Comment:
🔧 Let's make It slightly more descriptive. Something like `Zeebe BPMN Model`.

Reply:
👍